components/openindiana/ddu/Makefile

```diff
@@ -30,7 +31,7 @@ COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
 COMPONENT_SUMMARY=	Device Driver Utility (DDU)
 
 GIT=git
-GIT_REPO=https://github.com/OpenIndiana/ddu
+GIT_REPO=https://github.com/openindiana/ddu
```

Comment:
Why this change?  Organization name at github is OpenIndiana.

Reply:
This is an artifact of me using my own repo for the Source changes in DDU and then switching it back but all lowercase since it don't remember the casing offhand.

Reply:
Done